test/gui/shared/steps/account_context.py

```diff
@@ -163,6 +167,8 @@ def step(context, username, host):
     displayname = substituteInLineCodes(displayname)
     host = substituteInLineCodes(host)
 
+    # Q: recently added account is not focused
+    Toolbar.openAccount(displayname, host)
```

Comment:
Recently added account doesn't get the focus. The view is still in the old account. Is it expected behavior now?

CC @TheOneRing

Reply:
Thx for informing me about the regression could you please open an issue?

Reply:
here is the issue: https://github.com/owncloud/client/issues/11642